src/lib/bootloader/grub2.rb

```diff
@@ -155,8 +166,6 @@ def write_sysconfig(prewrite: false)
       prewrite ? sysconfig.pre_write : sysconfig.write
     end
 
-  private
```

Comment:
and same here. Write sysconfig have to be public method as it is used in installation.